flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/DefaultSchedulerFactory.java

```diff
@@ -49,6 +50,8 @@
  */
 public class DefaultSchedulerFactory implements SchedulerNGFactory {
 
+	private static final Logger LOG = LoggerFactory.getLogger(DefaultSchedulerFactory.class);
+
 	@Override
 	public SchedulerNG createInstance(
 			final Logger log,
```

Comment:
Should we use this logger?

Reply:
Actually I'm also thinking which is better: 
1. use `JobManager` logger for `DefaultScheduler` as we currently do, or 
2. create an individual logger for `DefaultScheduler`
WDYT?

Reply:
> Actually I'm also thinking which is better:
> 
>     1. use `JobManager` logger for `DefaultScheduler` as we currently do, or
> 
>     2. create an individual logger for `DefaultScheduler`
> 
> 
> WDYT?

At the moment, we use `JobManager` logger only for historical reasons, i.e., I did not want to change behavior. If we use static loggers per class, it would be easier to recognize which scheduler is used, and it would be slightly easier to find the line where a given log message is issued. However, I don't see immediate call of action.

Reply:
> Should we use this logger?

Using _this logger_ now.

Reply:
Ok.